src/Containers/App/App.js

```diff
@@ -4,19 +4,11 @@ import NotesSection from "../NotesSection/NotesSection"
 import NoteForm from "../NoteForm/NoteForm"
 import NotFound from "../../Components/NotFound/NotFound"
 import { connect } from "react-redux"
-import { setNotes, setLoading, setError } from "../../actions"
-import API from '../../utils/api';
+import { fetchNotes } from '../../thunks/fetchNotes'
 
 class App extends Component {
   async componentDidMount() {
-    const { setNotes, setLoading } = this.props
-    try {
-      const notes = await API.fetchData('notes', 'GET');
-      setNotes(notes)
-      setLoading(false)
-    } catch (error) {
-      setError(error)
-    }
+    this.props.fetchNotes()
```

Comment:
Awesome! I was thinking it would be good to use a thunk here because we are also call setNotes in the NoteForm after a note is created/edited. I did some editing in App to create a separate function for NoteForm but I can probably just edit it to use the thunk you created now.